eZ/Publish/Core/settings/repository/inner.yml

```diff
@@ -178,3 +179,28 @@ services:
     eZ\Publish\Core\Repository\Permission\LimitationService:
         arguments:
             $limitationTypes: !tagged_iterator { tag: ezpublish.limitationType, index_by: alias }
+
+    eZ\Publish\Core\Repository\Helper\RoleDomainMapper:
```

Comment:
Conflicting changes with #42. This was planned to be moved as 1st step before that. I prefer moving first, so we expose it in a correct place. I was going to finish that today.
Right now you have two instances of `RoleDomainMapper`. The first one for PermissionResolver and the other one for other services, injected via `Repository::getRoleDomainMapper`. This is why this needs to be handled first, separately.

Reply:
Note: #46 now includes this change from  #42. To be rebased after rebasing this one.